install/terraform/modules/gke-autopilot/cluster.tf

```diff
@@ -72,6 +73,13 @@ resource "google_container_cluster" "primary" {
     }
   }
 
+  dynamic "workload_identity_config" {
```

Comment:
Did you test this on Autopilot? I ask because I believe Workload Identity is always present on Autopilot: https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable